src/actions/entriesActions.js

```diff
@@ -0,0 +1,28 @@
+import entryRepo from '../repos/EntryRepo';
```

Comment:
this action is to do an async call to get entries for a specific period of time.